bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+# Set up Rails app. Run this script immediately after cloning the codebase.
+
+# Exit if any subcommand fails
+set -e
+
+# Set up ruby version file
+if [ ! -f .ruby-version ]; then
+  cp .ruby-version.example .ruby-version
+fi
+
+# Set up configurable environment variables
+if [ ! -f .env ]; then
+  cp config/env .env
+fi
+
+# Set up Ruby dependencies via Bundler
+gem install bundler --conservative
+bundle check || bundle install --path=vendor/bundle
```

Comment:
I can't think of any. If you don't hear back from @johnpaulashenfelter in a few days time, I would say go ahead and make the change.